srcpkgs/ncspot/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.11.2
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config python3"
+makedepends="openssl-devel dbus dbus-devel pulseaudio-devel"
```

Comment:
```suggestion
makedepends="openssl-devel dbus-devel pulseaudio-devel"
```

Reply:
Done

Reply:
None of my business but, why pulseaudio when you can build with alsa?

Reply:
That logic applies the other way as well. People who use pulse want to use it natively.

Reply:
Fair enough, I'll build it with cargo and install to `/usr/local/bin`.

Reply:
Btw, 0.12.0 was released 19 days ago.

Reply:
I don't see a reason for that, this package should support both pulse and alsa.